.azure-pipelines.yml

```diff
@@ -41,8 +41,8 @@ jobs:
     displayName: 'Set env'
 
   - script: |
-      python -m pip install --upgrade pip
-      pip install pytest-cov wheel
+      python -m pip install --upgrade pip || exit 1
+      pip install pytest-cov wheel || exit 1
```

Comment:
IDK if there’s a more elegant way to do this but as is now, the pipelines don’t make too much sense. See microsoft/azure-pipelines-tasks#10125

Reply:
Can we keep this these changes to a separate PR?

Anyways, black should be under it's own check, probably through pre-commit.

Reply:
Also, most of these tasks should fail if the earlier steps didn't complete right? For instance, if we fail to build wheels, check should fail.

Reply:
Steps yes, but individual steps will succeed if the last *command* happens to succeed.

Earlier commands will be ignored in determining the success of steps.

Reply:
Yeah, but that's just bash being bash, right?

It's kinda crazy they don't just have a "checked-script" task. However, I don't think adding `|| exit 1` after every command is better than adding `set -e` to the top of some commands – or splitting the command that's doing two checks into two tasks.